app/controllers/my-sessions/view.js

```diff
@@ -1,6 +1,7 @@
 import Controller from '@ember/controller';
 import { computed } from '@ember/object';
 import moment from 'moment';
+import { match } from '@ember/object/computed';
```

Comment:
@Anupam-dagar Match is defined but never used, Please check Travis Report, as one of the test cases is failing due to it